pallets/subtensor/runtime-api/src/lib.rs

```diff
@@ -14,6 +14,7 @@ sp_api::decl_runtime_apis! {
     pub trait NeuronInfoRuntimeApi {
         fn get_neurons(netuid: u16) -> Vec<u8>;
         fn get_neuron(netuid: u16, uid: u16) -> Vec<u8>;
+        fn get_neuron_certificate(netuid: u16, uid: u16) -> Vec<u8>;
```

Comment:
Here and every other method the returns a certificate should have a return type of `Option<...>` ( `Option<Vec<u8>>`, `Option<[u8; 32]>` if we go the array route). I'd prefer `Option<Certificate>` as we are defining a type anyway.